src/sections/Meshmap/meshmap.style.js

```diff
@@ -124,14 +124,33 @@ p.caption {
             display: block;
 
         }
+        .formRight select {
+          background: none repeat scroll 0 0 #FFFFFF;
+          border: 1px solid black;
+          border-radius: 10px;
+          height: 50px;
+          padding: 1rem 1rem;
+          width: 100%;
+          font-size: 0.85rem;
+          margin: 0.5rem 0;
+
+         
+          
+          background: url(http://cdn1.iconfinder.com/data/icons/cc_mono_icon_set/blacks/16x16/br_down.png) no-repeat right #ddd;
```

Comment:
Form is working now. Can we use a react icon instead of this external image here?

Reply:
For using the react icons I have to make changes in the dropdown list